src/bin/ion/commands/generate/templates/java/class.templ

```diff
@@ -39,6 +39,45 @@ import java.io.IOException;
         }
     {% endfor %}
 
+    public static class {{ model.name }}Builder {
```

Comment:
It's idiomatic to just call the inner class `Builder` so that it is `MyClass.Builder`.
```suggestion
    public static class Builder {
```